### PR TITLE
Add change log for 5.20.0

### PR DESCRIPTION
**Pull request by knabar**
Significant info here is the support for Django 3.2.19.

### CODE REVIEW

**Comment by knabar** [comment]
@jburel #466 is under "bug fixes", should it be in the "other changes" section?

**Comment by jburel** [comment]
Apologies I did not scroll enough and did not see it.
Looks good

**Review by jburel — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+5.20.0 (May 2023)
+-----------------
+
+## Other changes
+
+- Fix multiple file uploads and support Django 3.2.19 security update [#465](https://github.com/ome/omero-web/pull/465)
```
Comment (will-moore) [inline CHANGELOG.md:6]: Maybe "Fix muli-file upload validation and support... " since it's really the *validation* that was broken

**Comment by will-moore** [comment]
Looks good - just the one minor comment

**Review by will-moore — APPROVED** [review]

**Comment by jburel** [comment]
@knabar all PRs for the release have now been merged.
Are you okay to take care of it?

**Comment by knabar** [comment]
@jburel will work on the release now

**Comment by jburel** [comment]
thanks

**Pull request merged**